tests/tests-ui/cypress/integration/ProfilePage/cy_ProfilePage_Resolutions.js

```diff
@@ -0,0 +1,30 @@
+import {check_level} from '../../support/index'
+import {screen_sizes_dict} from '../../support/enums/sizes'
+
+describe('Profile page tests', () => {
+    beforeEach(() => {
+        //cy.logIn(true);
+    })
+
+    after(() => {
+        //cy.logOut();
+    })
```

Comment:
Is this needed? If not, remove the code

Reply:
it will be needed, I created the basic template for the tests after the functionality will come its going to be used